packages/mobile/README.md

```diff
@@ -498,6 +498,9 @@ We try to minimise the differences between running Valora in different modes and
 If you're having an error with installing packages, or `secrets.json` not existing:
 
 try to run `yarn postinstall` in the wallet root folder after running `yarn install`.
+  
+If some of your assets are not loaded and you see an error running sync_branding.sh.
+Check if you have set up your Github connection with SSH.
```

Comment:
maybe would be nice to add a reference for how to do this? i found this article really helpful when i was doing this for the first time https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account :)